### PR TITLE
License, badges and dependencies

### PR DESCRIPTION
**Pull request by asbjornu**
Update license, badges and dependencies.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# Contributor Covenant Code of Conduct
```
Comment (payex-dev) [inline CODE_OF_CONDUCT.md:1]: Codacy Issue found: [Could not find module `remark-slug`](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5624734)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5624734&bid=18425233)

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (NullableInt) [reply CODE_OF_CONDUCT.md:1]: Remove `- "remark-slug"` from `.remarkrc.yaml` aprox linje 3. And run `npm rm remark-slug` to solve this.

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5624734&bid=18425233)

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (payex-dev) [inline CODE_OF_CONDUCT.md:1]: Codacy Issue found: [Could not find module `remark-slug`](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5624734)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5624734&bid=18425233)

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (payex-dev) [inline CODE_OF_CONDUCT.md:1]: Codacy Issue found: [Could not find module `remark-slug`](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5624734)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5624734&bid=18425233)

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (payex-dev) [inline CODE_OF_CONDUCT.md:1]: Codacy Issue found: [Could not find module `remark-slug`](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5624734)

**Pull request merged**